src/components/atomic/Tag.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { HTMLAttributes } from "react";
+import classNames from "classnames";
+
+type TagProps = HTMLAttributes<HTMLDivElement>;
+
+const Tag = ({ className, ...props }: TagProps) => {
+  const styles = classNames({
```

Comment:
Might be good to reference this styling for the tag component: [all mentors figma page](https://www.figma.com/file/qfzd9rev3yhzKeCZBv8lIB/wireframe?node-id=3370%3A4518)

2nd note: The edges have standardly been rounded-md I believe

Reply:
Thank you I totally missed that on the figma my b